lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_playbook_method.rb

```diff
@@ -158,5 +158,16 @@ def api_token
     def api_url
       @api_url ||= MiqRegion.my_region.remote_ws_url
     end
+
+    def miq_request_task_url
+      result = {}
+      if @workspace.root['vmdb_object_type']
+        task_obj = @workspace.root[@workspace.root['vmdb_object_type']]
```

Comment:
The name could be misleading since we do not know it is a task_object.  `vmdb_object` would be a better name.